package.yaml

```diff
@@ -100,6 +102,9 @@ dependencies:
 library:
   source-dirs: library
 
+  generated-exposed-modules:
+    - Paths_ipfs
```

Comment:
`Paths_ipfs`?

Reply:
Another thing where `stack sdist` was freaking out at me 
```
Packages using 'cabal-version: 2.0' and the autogenerated module Paths_* must include it also on the 'autogen-modules' field besides 'exposed-modules' and 'other-modules'. This specifies that the module does not come with the package and is generated on setup. Modules built with a custom Setup.hs script also go here to ensure that commands like sdist don't fail.
```
`generated-exposed-modules` adds translates to `autogen-modules` in cabal land

Frankly not sure what `Paths_*` modules are so if you have any wisdom on that, would love to know

Reply:
IIRC it's generated by Cabal during compilation. Not sure why it's asking for it explicitly 🤷‍♀